pkg/controller/sparkapplication/controller.go

```diff
@@ -503,6 +503,10 @@ func (c *Controller) syncSparkApplication(key string) error {
 			appToUpdate = c.submitSparkApplication(appToUpdate)
 		}
 	case v1beta1.SucceedingState:
+		if appToUpdate.Status.ExecutionAttempts > 1 {
```

Comment:
This check should be `>=1` . I don't think we even need this check given that we are in `SuceedingState`

Reply:
We don't need to check. Fixed.